.github/USING_FEEDBACK_DATA.md

```diff
@@ -0,0 +1,198 @@
+
+```dm
+SSblackbox.record_feedback("nested tally", "example", 1, list("fruit", "orange", "apricot"))
+SSblackbox.record_feedback("nested tally", "example", 2, list("fruit", "orange", "orange"))
+SSblackbox.record_feedback("nested tally", "example", 3, list("fruit", "orange", "apricot"))
+SSblackbox.record_feedback("nested tally", "example", 10, list("fruit", "red", "apple"))
+SSblackbox.record_feedback("nested tally", "example", 1, list("vegetable", "orange", "carrot"))
+```
+
+Will produce the following JSON:
+
+```json
+{
+    "data":{
+        "fruit":{
+            "orange":{
+                "apricot":4,
+                "orange":2
+            },
+            "red":{
+                "apple":10
+            }
+        },
+        "vegetable":{
+            "orange":{
+                "carrot":1
+            }
+        }
+    }
+}
+```
+
+#### NOTE
+
+Tracking values associated with a number can't merge with a nesting value, trying to do so will append the list
+
+```dm
+SSblackbox.record_feedback("nested tally", "example", 3, list("fruit", "orange"))
+```
+
+Will produce the following JSON:
+
+```json
+{
+    "data":{
+        "fruit":{
+            "orange":{
+                "apricot":4,
+                "orange":2
+            },
+            "red":{
+                "apple":10
+            },
+            "orange":3
+        },
+        "vegetable":{
+            "orange":{
+                "carrot":1
+            }
+        }
+    }
+}
+```
+
+Avoid doing this, since having duplicate keys in JSON (data.fruit.orange) will break when parsing.
+
+### Tally
+
+`tally` is used to track the number of occurances of multiple related values (EG: how many times each type of gun is fired). Further calls to the same key will add or subtract from the saved value of the data key if it already exists, and append the key and it's value if it doesn't exist.
+
+```dm
+SSblackbox.record_feedback("tally", "example", 1, "sample data")
+SSblackbox.record_feedback("tally", "example", 4, "sample data")
+SSblackbox.record_feedback("tally", "example", 2, "other data")
+```
+
+Will produce the following JSON:
+
+```json
+{
+    "data":{
+        "sample data":5,
+        "other data":2
+    }
+}
+```
+
+### Text
+
+`text` is used for simple single-string records (EG: the current map). Further calls to the same key will append saved data unless the overwrite argument is true or it already exists. When encoded, calls made with overwrite will lack square brackets
+
+```dm
+SSblackbox.record_feedback("text", "example", 1, "sample text")
+SSblackbox.record_feedback("text", "example", 1, "other text")
+```
+
+Will produce the following JSON:
+
+```json
+{
+    "data":[
+        "sample text",
+        "other text"
+    ]
+}
+```
+
+## Feedback Versioning
+
+If the logging format (IE: What data is logged) for a variable is ever changed, the version needs bumping. This can be done with the `versions` list on the subsystem definition itself. All values default to `1`.
```

Comment:
```suggestion
If the logging format (i.e.: What data is logged) for a variable is ever changed, the version needs bumping. This can be done with the `versions` list on the subsystem definition itself. All values default to `1`.
```